base/float.jl

```diff
@@ -199,6 +199,17 @@ promote_rule(::Type{Float64}, ::Type{Float32}) = Float64
 widen(::Type{Float16}) = Float32
 widen(::Type{Float32}) = Float64
 
+promote_op{T<:Union{Float32,Float64}}(::typeof(trunc), ::Type{Signed}, ::Type{T}) = Int
```

Comment:
Sorry, I completely misread that line due to what I was working on at the same time, carry on.